lib/iterator.js

```diff
@@ -199,6 +199,66 @@ class Iterator {
     return result + suffix;
   }
 
+  // Find value in this iterator by comparing every value with
+  // the found one using `comparator`
+  // Signature: comparator[, accessor[, thisArg]]
+  //   comparator <Function> returns true if new value should be accepted
+  //     currValue <any> current value, starts with undefined
+  //     nextValue <any> next value
+  //     Returns: <boolean> true if next value should be accepted
+  //   accessor <Function> gets value to compare by, current iterator value
+  //       is used by default
+  //     value <any> current iterator value
+  //     Returns: <any> value to compare by
+  //   thisArg <any> value to be used as `this` when calling `accessor` and
+  //       `comparator`
+  // Returns: last iterator value where `comparator` returned `true`,
+  //     `undefined` by default
```

Comment:
```suggestion
  //     <undefined> by default
```

Reply:
I assume these are to get them to be made into links but I think it would be better to just handle `` `type` `` the same way as ` <type> `